sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -254,6 +251,21 @@ class SparkSqlAstBuilder extends AstBuilder {
     }
   }
 
+   /**
+    * A column path can be specified as an parameter to describe command. It is a dot separated
+    * elements where the last element can be a String.
+    * TODO - check with Herman
```

Comment:
It is a bit more complicates than I thought. We allow strings here because Hive allows us to use the `'$elem'`, `'$keys'` and `'$values'` 'keywords'. That is why I added strings to the rule. I am not sure if we should support this. What do you guys think?

This is what I found in the Hive manual:
```SQL
DESCRIBE [EXTENDED|FORMATTED]  [db_name.]table_name[ col_name ( [.field_name] | [.'$elem$'] | [.'$key$'] | [.'$value$'] )* ];
```

See also: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Describe